models/snapchat.yml

```diff
@@ -151,6 +151,11 @@ models:
         description: The current used by the account in Snapchat.
       - name: base_url
         description: The base URL of the ad, extracted from the web_view_url.
+        tests:
+          - not_null:
+              config:
+                # Below is so we don't run this test if the user has allowed urls to have null entries
+                enabled: "{{ var('ad_reporting__url_report__using_null_filter', True) }}"
```

Comment:
i believe this config needs to also appear in the unique combo test as well

Reply:
updated.